### PR TITLE
Popup

### PR DESCRIPTION
**Pull request by IrynaY**
### HTML & CSS practice: Google-style Popup

[Demo](https://irynay.github.io/html-css-popup/) |
[Code base](https://github.com/IrynaY/html-css-popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/IrynaY/html-css-popup/style.css [hunk 1]
```diff
@@ -1,12 +1,95 @@
...
+  background-image: url(https://lh3.googleusercontent.com/-egvmId9n5mM/AAAAAAAAAAI/AAAAAAAAAAA/ACevoQPKMChGX8tFI9H3ERdiAH6-rYbcLg/s32-c-mo/photo.jpg);
+  background-position: -261px -33px;
+  border-radius: 50px;
+}
+
+.popup {
+  display: none;
+  position: absolute;
+  width: 320px;  
+  /* max-height: 80%; */
```
Comment (A-Ostrovnyy) [inline submissions/IrynaY/html-css-popup/style.css]: Don't commit commented lines or blocks of code.

File: submissions/IrynaY/html-css-popup/style.css [hunk 2]
```diff
@@ -1,12 +1,95 @@
...
+  right: 25px;
+  top: 80px;
+  background: #fff;
+  border: 1px solid #ccc;
+  box-shadow: 0 0 1em #ccc;
+}
+
+.popup ul{
```
Comment (A-Ostrovnyy) [inline submissions/IrynaY/html-css-popup/style.css]: Avoid styling by HTML tag.

File: submissions/IrynaY/html-css-popup/style.css [hunk 3]
```diff
@@ -24,11 +24,25 @@ body{
   margin: 0;
 }
 
-.hide:checked ~ .popup {
+#show-apps:checked, #show-apps:checked, #show-all {
```
Comment (A-Ostrovnyy) [inline submissions/IrynaY/html-css-popup/style.css]: Avoid styling by id.

File: submissions/IrynaY/html-css-popup/style.css [hunk 4]
```diff
@@ -24,11 +24,25 @@ body{
...
+  width: 0;
+  height: 0;
+}
+
+#show-apps:checked ~ .popup, #show-all:checked ~ .view-more, #show-all, #show-apps:checked ~ .reset{
```
Comment (A-Ostrovnyy) [inline submissions/IrynaY/html-css-popup/style.css]: Easier to read selector if each of them started from the new line.

File: submissions/IrynaY/html-css-popup/style.css [hunk 5]
```diff
@@ -68,28 +90,125 @@ body{
...
+
+.more label, .menu-righ li:hover, .menu-right li:hover, .menu-right li input:hover{
+  cursor: pointer;
+}
+
+.menu-right li:hover, .menu-right li input:hover{
+  text-decoration: underline;
+}
+
+.app-list li:nth-child(1) .app-list-img{
```
Comment (A-Ostrovnyy) [inline submissions/IrynaY/html-css-popup/style.css]: Better use `:first-child` for first element.

File: submissions/IrynaY/html-css-popup/style.css (hunk 3 above)
Comment (IrynaY) [reply submissions/IrynaY/html-css-popup/style.css]: Thanks for the review. I left the item with  id #show-all, as there is a label to it. Its OK?

Comment (A-Ostrovnyy) [reply submissions/IrynaY/html-css-popup/style.css]: I prefer to separate responsibility: `.class` for style, `#id` - js/link for labels etc. 
On the future, it's up to you.

**Pull request merged**